### PR TITLE
Add js files And stylings in todolist

### PR DESCRIPTION
**Pull request by Sahar-SE**
### This PR contains:

- All necessary functionalities which is mentioned in project.
- It can add a task.
- It can remove completed tasks.
- It contains a Linter check points and a README file.

### CODE REVIEW

**Review by brytebee — APPROVED** [review]
Hi, @Sahar-SE👋, 

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Sahar-SE, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+Javascript
+Sublime Text
+
+# Author
+👤 Sahar Saba Amiri
+
+Gmail: saharsaba.amiri123@gmail.com
+LinkedIn: https://www.linkedin.com/sahar-saba-amiri
+GitHub : https://github.com/Sahar-SE
+
```
Comment (roshan-bajgain) [inline README.md:16]: - [ ] [Optional] Kindly write the step to `clone, install and run the app` on the browser So any user who wants to see your project can run it by looking at the steps and I would suggest you provide the `live demo` on your readme file So in the future recruiters can get access to your project easily which create positive impact on the job.

**Review by roshan-bajgain — APPROVED** [review]
Hi @Sahar-SE, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️
- Your project works properly good job. ✔️ 

## Optional suggestions

_Check the comment under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**